docs/Data correction documentary-cleared.ipynb

```diff
@@ -0,0 +1,301 @@
+    "\n",
+    "\\begin{equation}\n",
+    "\\begin{pmatrix} I^{++}_{t,\\lambda} \\\\ I^{+-}_{t,\\lambda}  \\\\ I^{-+}_{t,\\lambda} \\\\  I^{--}_{t,\\lambda}\\end{pmatrix} = \\hat{T}^a(t,\\lambda)*\\hat{T}^p(t,\\lambda) \\\\\n",
+    "\\begin{pmatrix} S^{++}_{t,\\lambda} \\\\ S^{+-}_{t,\\lambda}  \\\\ S^{-+}_{t,\\lambda} \\\\  S^{--}_{t,\\lambda}\\end{pmatrix}\n",
+    "\\end{equation}\n",
+    "\n",
+    "\n",
+    "\n",
+    "Both $\\hat{T^p}$ and $\\hat{T^a}$ can be derived from their cell parameters (the transmission of the empty cell $T^{g, p|a}$,  and the cell opacity $O$), and the time-dependent nuclear polarization of the cell $P_{3He}(t)$.<br>\n",
+    "This general method for time-dependent $^3$He-cells can be adapted for other use-cases such as using supermirrors or in-situ polarized 3He-cells, see \"Note\" at the end of the page. \n",
+    "\n",
+    "First, the created test-data needs to be readout and grouped by purpose, e.g., if it describes a sample-run, a direct-beam-run, or a background measurement without cells and sample. All data will be reduced (PROBABLY BEAMLINE SPECIFIC?) using the normalization term described in https://scipp.github.io/ess/techniques/sans/sans-beam-center-finder.html. Second, for the calculation of time-dependent $P_{3He}(t)$, the ROI's of the direct-beam region and background region have to be defined (USING BEAMCENTER FINDER OPTION OF SANS2D?). Third, we will describe the data correction workflow with a calculation of $O$, $P_{3He}(t)$, and $T^{\\pm}$ of the polarizer and analyzer, and the application on the sample-data.\n",
+    "\n",
+    "To facilitate an in-situ data correction during the experiments, we have implemented two different approaches: (i) a rough data-correction based on a $^3$He-cell opacity calculated by its cell parameters and filling pressure (\"workflow 1\") which can be used in-situ during the experiment, and (ii) a precise calculation based on the calculation of opacity by a direct-beam measurement on the depolarized cell at its end of lifetime (i.e., end of cell-usage for this experiment) (\"workflow 2\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "405b0f3e-67ab-43d8-80de-566a1b93785a",
+   "metadata": {},
+   "source": [
+    "### 4. Data correction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "add255a3-3222-444a-b2e6-b8cef3a8b2db",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "#### 4.1 Calculation of $^3$He opacity \n",
+    "\n",
+    "$\\mathbf{Workflow 1}$:<br>\n",
+    "The $^3$He opacity is a cell-specific parameter and can vary after each filling process. It is given by the cell pressure $p$ at 20$\\degree$, the path length of neutron through the cell $l$, and the neutron wavelength $\\lambda$ (Ref 1):\n",
+    "\n",
+    "$$O^{p|a}(\\lambda)= (0.0733 * p^{p|a} * l^{p|a} )\\lambda$$\n",
```

Comment:
I have to add newlines, like
```
$$
math here
$$
```
otherwise I get rendering errors. Can you add those everywhere?

Reply:
Sure! Strange the notebook options seem a bit different for me